src/main.ts

```diff
@@ -295,6 +321,24 @@ function handleKeyDown({ key }: KeyboardEvent): void {
   }
 }
 
+async function handleFindTrack(): Promise<void> {
+  // Load track value
+  const trackId = trackInput.valueAsNumber;
+  const newTrack = dataset!.buildTrack(trackId);
+
+  // Check for track validity
+  if (newTrack.length() > 0) {
+    selectedTrack = newTrack;
+    timeControls.setCurrentFrame(selectedTrack.times[0]);
```

Comment:
Do we know if users will ever want to select a track and remain on the current frame, e.g. to see the position of several tracks at a time? Not a blocking concern for this PR, just curious

Reply:
good question, I'd say we would wait until they request it.  The main use case is : "someone told me to look at track X"